README.md

```diff
@@ -73,31 +73,44 @@ Note that unlike `JSON.stringify()`, object property ordering is not considered.
 
 A re-export of `faker` from [`@faker-js/faker`](https://github.com/faker-js/faker). We do not alter faker in any way, and do not seed it.
 
-### `copycat.oneOf(input, values)`
+### `copycat.scramble(string[, options])`
 
-Takes in an [`input`](#input) value and an array of `values`, and returns an item in `values` that corresponds to that `input`:
+Takes in a `string` value, and returns a string with the same length, but with each character replaced with a different character in the same character range:
+* By default, spaces are preserved (see `preserve` option below)
+* Lower case ascii characters are replaced with lower case ascii letters
+* Upper case ascii characters are replaced with upper case ascii letters
+* Digits are replaced with digits
+* Any other ascii character in the code point range `32` to `126` (`0x20 - 0x7e`) is replaced with an ascii in the same range
+* Any other character is replaced with a [Latin-1](https://en.wikipedia.org/wiki/Latin-1_Supplement) character in the range of (`0x20 - 0x7e`, or `0xa0 - 0xff`)
 
 ```js
-copycat.oneOf('foo', ['red', 'green', 'blue'])
-// => 'red'
+import { copycat } from '@snaplet/copycat'
+
+copycat.scramble('Zakary Hessel')
+// => 'Wradls Kbicbs'
 ```
 
-### `times(input, range, fn)`
+#### `options`
 
-Takes in an [`input`](#input) value and a function `fn`, calls that function repeatedly (each time with a unique input) for a number of times within the given `range`, and returns the results as an array:
+- **preserve**: An array of characters that should remain the same if present in the given input string
 
 ```js
-copycat.times('foo', [4, 5], copycat.word)
-// => [ 'Raeko', 'Vame', 'Kiyumo', 'Koviva', 'Kiyovami' ]
+import { copycat } from '@snaplet/copycat'
+
+copycat.scramble('foo@bar.org', { preserve: ['@', '.'] })
+// => 'oxb@fmc.ahs'
```

Comment:
One limitation to this api, is that we'd be preserving chars we may not want to preserve. e.g. for emails:

```
> copycat.scramble('foo.bar@example.org', { preserve: ['@','.'] })
'afg.mmo@wxigbcw.rti'
```

We may not want to to preserve that first `.` (it's revealing something about the original value).

Not sure if its a deal-breaker or not, but thought worth mentioning.

Reply:
As I started reading it, I thought it might make sense to not preserve the '.' in the first part of the email address, but then I recalled that most email providers ignore that anyway.  I don't think it's an issue that we need to worry about.  Scrambling is not obfuscating entirely.

Reply:
> but then I recalled that most email providers ignore that anyway.

Ooh, true

> I don't think it's an issue that we need to worry about. Scrambling is not obfuscating entirely.

Great point!